docker/build/devdeps.ompi.Dockerfile

```diff
@@ -36,10 +38,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 ENV PMI_INSTALL_PREFIX=/usr/local/pmi
 RUN mkdir -p /var/tmp && wget -q -nc --no-check-certificate -P /var/tmp https://download.schedmd.com/slurm/slurm-21.08.8.tar.bz2 \
     && tar -x -f /var/tmp/slurm-21.08.8.tar.bz2 -C /var/tmp -j && cd /var/tmp/slurm-21.08.8 \
-    &&  CC=gcc CFLAGS="$COMMON_COMPILER_FLAGS" \
-        CXX=g++ CXXFLAGS="$COMMON_COMPILER_FLAGS" \
-        F77=gfortran F90=gfortran FFLAGS="$COMMON_COMPILER_FLAGS" \
-        FC=gfortran FCFLAGS="$COMMON_COMPILER_FLAGS" \
+    && export common_flags=$([ "$TARGETARCH" == "arm64" ] && echo "$COMMON_COMPILER_FLAGS_ARM" || echo "$COMMON_COMPILER_FLAGS") \
+    &&  CC=gcc CFLAGS="$common_flags" \
+        CXX=g++ CXXFLAGS="$common_flags" \
```

Comment:
Will this use g++ for the clang++ builds as well?

Reply:
Yes. The entire Open MPI + dependencies build is using the same compiler that the used CUDA version uses as host compiler, specifically, gcc 11. This goes back to the discussion regarding which compiler is used for which components.